tests/e2e/vmservice_vm.go

```diff
@@ -263,6 +272,7 @@ var _ bool = ginkgo.Describe("[vmsvc] vm service with csi vol tests", func() {
 			err = e2eVSphere.waitForCNSVolumeToBeDeleted(staticPv.Spec.CSI.VolumeHandle)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
+
```

Comment:
Here, suppose if vm creation fails in case of latebinding mode, then volHandle will be nil and test might panic. Please take a look here

Reply:
fixed